### PR TITLE
Use matrix-org/pr-details-action@v1.1 for Netlify deploys

### PR DESCRIPTION
**Pull request by t3chguy**
Preview: https://pr1347--matrix-org-previews.netlify.app

### CODE REVIEW

File: .github/workflows/netlify.yml [hunk 1]
```diff
@@ -53,7 +43,7 @@ jobs:
           deploy-message: "Deploy from GitHub Actions"
           enable-pull-request-comment: false
           enable-commit-comment: false
-          alias: pr${{ steps.readctx.outputs.prnumber }}
+          alias: pr${{ steps.prdetails.outputs.pr_id }}
```
Comment (richvdh) [inline .github/workflows/netlify.yml:46]: this is maybe more of a comment on the action, but this doesn't seem to be defined in the action's documentation. I'd really rather the action be properly documented before we switch things over to it.

**Review by richvdh — CHANGES_REQUESTED** [review]

File: .github/workflows/netlify.yml (hunk 1 above)
Comment (t3chguy) [reply .github/workflows/netlify.yml:46]: Good call, I tend to rely on documenting within actions.yml but duplicating to README helps - https://github.com/matrix-org/pr-details-action/blob/main/README.md#inputs

**Comment by thibaultamartin** [comment]
@t3chguy same here, this one can safely be closed I presume?

**Comment by thibaultamartin** [comment]
Err, got a bit trigger happy here, sorry. Please reopen if needed

**Pull request closed without merging**